spec/lib/process_settings/targeted_settings_spec.rb

```diff
@@ -22,9 +22,14 @@
 
   describe "#initialize" do
     it "requires version:" do
+      expected_message = if RUBY_VERSION < '2.7'
+        /missing keyword: version/
+      else
+        /missing keyword: :version/
```

Comment:
That's odd. Seems like a step backwards that newer Ruby is revealing that keywords are implemented with symbols? Oh well.